src/core/agent/services/credentialService.types.ts

```diff
@@ -6,6 +6,10 @@ enum CredentialStatus {
   REVOKED = "revoked",
 }
 
+enum ConnectionNoteKeys {
```

Comment:
This should be in `connectionService.types.ts`, and I meant anything related to a connection, so more like:

```
enum KeriaContactKeyPrefix {
  CONNECTION_NOTE = "note:",
}
```